src/sentry/static/sentry/app/components/organizations/homeSidebar.jsx

```diff
@@ -137,7 +137,18 @@ const HomeSidebar = createReactClass({
           >
             {t('Dashboard')}
           </ListLink>
-          <ListLink to={`${pathPrefix}/teams/`}>{t('Projects & Teams')}</ListLink>
+
+          {!features.has('internal-catchall') && (
+            <ListLink to={`${pathPrefix}/projects/`}>{t('Projects & Teams')}</ListLink>
```

Comment:
hmm wait was changing this to the `projects` link intentional?

Reply:
Yeah I think we want projects as default